client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -45,7 +46,14 @@ describe('CodeEvalResultObjectFactory', function() {
   describe('getCorrectnessTestResults', function() {
     it('should correctly get correctness test results', function() {
       expect(codeEvalResult.getCorrectnessTestResults())
-      .toMatch(CORRECTNESS_TEST_RESULTS);
+      .toEqual(CORRECTNESS_TEST_RESULTS);
+    });
+  });
+
+  describe('getLastTaskResults', function() {
+    it('should correctly get last task results', function() {
+      expect(codeEvalResult.getLastTaskResults())
+      .toEqual(LAST_TASK_RESULTS);
```

Comment:
Wasn't this in a different branch? You should either remove this one here or remove the other PR?